crates/metaslang/bindings/src/lib.rs

```diff
@@ -22,23 +23,40 @@ type GraphHandle = stack_graphs::arena::Handle<stack_graphs::graph::Node>;
 pub struct Bindings<KT: KindTypes + 'static> {
     graph_builder_file: File<KT>,
     functions: Functions<KT>,
-    stack_graph: StackGraph,
+    pub stack_graph: StackGraph,
```

Comment:
I suggest making this private, since it is not used outside this module:

```suggestion
    stack_graph: StackGraph,
```

Reply:
Oh, good catch. This is a leftover from an old test that would render the stack graph as Mermaid. I'll change this.